src/mon/PGMap.cc

```diff
@@ -891,18 +897,23 @@ void PGMapDigest::dump_object_stat_sum(
       f->dump_int("rd_bytes", sum.num_rd_kb * 1024ull);
       f->dump_int("wr", sum.num_wr);
       f->dump_int("wr_bytes", sum.num_wr_kb * 1024ull);
-      f->dump_int("raw_bytes_used", sum.num_bytes * raw_used_rate * curr_object_copies_rate);
```

Comment:
@ifed01 this is a user-visible change.  and this change breaks all tests which assumes that we have `raw_bytes_used` in the output of `ceph df detail --format=json`, for instance, http://pulpito.ceph.com/kchai-2018-09-16_14:12:08-rados-wip-kefu-testing-2018-09-14-0938-distro-basic-mira/3029785/ and http://pulpito.ceph.com/kchai-2018-09-16_14:12:08-rados-wip-kefu-testing-2018-09-14-0938-distro-basic-mira/3029793/ .

if we decided to have this non-backward-compatible change, could you please update the tests and note it in the `PendingReleaseNotes` ?